cmd/cmd.go

```diff
@@ -144,7 +144,11 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 		if resp.Digest != "" {
 			bar, ok := bars[resp.Digest]
 			if !ok {
-				bar = progress.NewBar(fmt.Sprintf("pulling %s...", resp.Digest[7:19]), resp.Total, resp.Completed)
+				msg := resp.Status
```

Comment:
should this be in a separate change?

Reply:
I needed this to achieve the UX in the example above with a progress bar and size
```
quantizing F16 model to Q4_0 100% ▕████████████████████████████████████▏  68 GB
```

Without that, it says `pulling <digest> 100% ▕████████████████████████████████████▏  68 GB`